asf_heat_pump_suitability/pipeline/run/heat_network_city_centres.py

```diff
@@ -0,0 +1,106 @@
+"""
+Contains script to add boolean flags to label residential UPRNs indicating whether they are:
+- located within a heat network zone
+- located within a city centre
+
+To run the script:
+python asf_heat_pump_suitability/pipeline/run/heat_network_city_centres.py
+
+Set the optional `local_authorities` parameter to `plymouth`, `plymouth_similar`, or `sampling_areas`.
+Set to `plymouth` to run for Plymouth Local Authority; `plymouth_similar` to run for Plymouth plus four other similar
+Local Authorities (Liverpool, Portsmouth, Southampton, Swansea); 'sampling_areas' to run for Plymouth plus five other
+Local Authorities for sampling buildings (Bath, Bradford, Glasgow, Manchester, Nottingham); or do not use to run for all
+of Great Britain.
+"""
+
+import argparse
+
+
+def parse_arguments() -> argparse.Namespace:
+    """
+    Create ArgumentParser and parse.
+
+    Returns:
+        argparse.Namespace: populated `Namespace`
+    """
+    parser = argparse.ArgumentParser()
+
+    # Placeholder - this arg is to look up correct paths for residential UPRNs and heat network dataset
+    parser.add_argument(
+        "--local_authorities",
+        help="Run script for either all of Great Britain; Plymouth only {plymouth}; or Plymouth and 4 similar local authorities {plymouth_similar}; or Plymouth and 5 different local authorities {sampling_areas}. Default to all of GB",
+        type=str,
+        default="GB",
+        required=False,
+    )
+
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    import polars as pl
+
+    from asf_heat_pump_suitability import config
+    from asf_heat_pump_suitability.getters import base_getters, get_datasets
+    from asf_heat_pump_suitability.pipeline.prepare_features import lat_lon
+    from asf_heat_pump_suitability.pipeline.transform import heat_network_zones
+    from asf_heat_pump_suitability.utils import save_utils
+
+    args = parse_arguments()
+
+    if args.local_authorities.lower() == "plymouth":
+
+        # Load residential OS UPRNs in Plymouth
+        print("Loading residential UPRN dataset for Plymouth Local Authority...")
+        uprn_df = base_getters.load_df_from_s3(
+            config["data"]["processed"]["plymouth_residential_uprns"]
+        )
+        uprn_gdf = lat_lon.generate_gdf_uprn_coords(uprn_df)
```

Comment:
```suggestion
        uprn_gdf = uprns.generate_gdf_uprn_coords(uprn_df)
```
What you have isn't incorrect, but will be removed at some point. I've replaced with the same function from a different file. The `lat_lon.py` file will be removed during the transition from v1.0.0 to v2.0.0.